src/databricks/azext_databricks/_params.py

```diff
@@ -23,17 +22,9 @@ def load_arguments(self, _):
         c.argument('tags', tags_type)
         c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
         c.argument('managed_resource_group', help='The managed resource group to create. It can be either a name or a resource ID.')
-        c.argument('aml_workspace_id', help='Azure Machine Learning workspace guid.')
-        c.argument('custom_virtual_network_id', options_list=['--virtual-network'], help='Virtual network name or resource ID.')
-        c.argument('custom_public_subnet_name', options_list=['--public-subnet-name'], help='The name of a new public subnet.')
-        c.argument('custom_private_subnet_name', options_list=['--private-subnet-name'], help='The name of a new private subnet.')
-        c.argument('enable_no_public_ip', arg_type=get_three_state_flag(), help='Set true to disallow public IP. Default is false')
-        c.argument('load_balancer_backend_pool_name', help='The name of load balancer backend pool name to create.')
-        c.argument('load_balancer_id', options_list=['--load-balancer'], help='Load balancer name or resource ID.')
-        c.argument('relay_namespace_name', help='The name of relay namespace. Default value is "dbrelay" as prefix, followed by random digits and letters.')
-        c.argument('storage_account_name', help='Storage account name to create. Default value is "dbstorage" as prefix, followed by random digits and letters.')
-        c.argument('storage_account_sku_name', options_list=['--storage-account-sku'], arg_type=get_enum_type(['Standard_LRS', 'Standard_ZRS', 'Standard_GRS', 'Standard_RAGRS', 'Premium_LRS']), default='Standard_GRS', help='Storage account SKU.')
-        c.argument('vnet_address_prefix', default='10.139', help='The vnet address prefix.')
+        c.argument('custom_virtual_network_id', options_list=['--vnet'], arg_group='Custom VNET', help='Virtual network name or resource ID.')

```

Comment:
do you want to comment them out for future each pick up?

Reply:
We can rely on this PR to put them back, keep it clean for this release.